public/app/percona/shared/services/services/Services.types.ts

```diff
@@ -10,25 +12,36 @@ export enum ServiceType {
   external = 'EXTERNAL_SERVICE',
 }
 
+export enum ServiceStatus {
+  UP = 'UP',
+  DOWN = 'DOWN',
+  UNKNOWN = 'UNKNOWN',
+  NA = 'N/A',
+}
+
 export interface ListServicesBody {
   node_id: string;
   service_type: ServiceType;
   external_group: string;
 }
 
 export interface DbServicePayload {
+  service_type: Databases | 'external';
   service_id: string;
   service_name: string;
   node_id: string;
+  node_name: string;
+  status?: ServiceStatus | 'STATUS_INVALID';
   enviroment?: string;
   cluster?: string;
   replication_set?: string;
   custom_labels?: Record<string, string>;
+  agents?: Array<{ agent_id: string; status?: ServiceAgentStatus; is_connected?: boolean }>;
```

Comment:
Could you create a separate type for agent items, it might get messy if there will be additional props in the future.

Reply:
done 👍